src/api/resources/base.py

```diff
@@ -446,6 +448,30 @@ def remove_invalid_fields(self, queryset, fields, view, request):
         ]
 
 
+class SafeSearchFilter(SearchFilter):
+
+    def _check_search_terms(self, search_fields, search_terms):
+        if (
+            not search_fields
+            or not search_terms
+            or "$" not in [sf[0] for sf in search_fields]
```

Comment:
Is this the only thing wrong with the test query? Is there anything else we should be protecting against?

Reply:
I wanted to address the immediate need it can always built out (more safety)